core/context-strategy/autoscale-context.ts

```diff
@@ -242,6 +251,9 @@ export class ConstantIntervalHeartbeatSyncStrategy implements HeartbeatSyncStrat
             this.result = HealthCheckResult.OnTime;
             // no old next heartbeat time for the first heartbeat, use the current arrival time.
             oldNextHeartbeatTime = heartbeatArriveTime;
+            // no old record for reference, use the seq provided by the device. If the seq is
+            // NaN, it means no sequence provided by the device, then use 0.
+            oldSeq = (isNaN(deviceSyncInfo.sequence) && 0) || deviceSyncInfo.sequence;
```

Comment:
`(isNaN(NaN) && 0) || undefined` -> `undefined`

you should use a ternary operator instead

`isNaN(deviceSyncInfo.sequence) ? 0 : deviceSyncInfo.sequence`

Reply:
fixed